starter-code/src/app/contact-list/contact-list.component.html

```diff
@@ -15,4 +25,4 @@
   <input type="text" [(ngModel)]="newContact.image">
 </div>
 
-<button (click)="addContact">Submit New Contact</button>
+<button (click)="addContact(newContact)">Submit New Contact</button>
```

Comment:
No hace falta pasarle el newContact, lo declara en TS y los recoges atrás con THIS. Si no ves esto que digo ven a preguntarme.